sdk/src/main/java/io/horizen/account/api/rpc/types/EthereumTransactionView.java

```diff
@@ -79,5 +79,38 @@ public EthereumTransactionView(EthereumTransaction tx, EthereumReceipt receipt,
         blockNumber = BigInteger.valueOf(receipt.blockNumber());
         transactionIndex = BigInteger.valueOf(receipt.transactionIndex());
     }
+
+    // constructor used for the pending transactions
+    public EthereumTransactionView(EthereumTransaction tx) {
```

Comment:
I see a lot of code duplication between this constructor and the one above for inchain txs.
the difference is only in `EthereumReceipt receipt, BigInteger baseFee`.
Let's unify the code of this 2 constructors.